keyboards/gray_studio/hb85/keymaps/via/config.h

```diff
@@ -0,0 +1,3 @@
+#pragma once
```

Comment:
Can we get a license header here, please?

Reply:
done